metadatahandler.py

```diff
@@ -1,16 +1,80 @@
+import collections
+from enum import Enum
 import os
+import platform
+import util
 import shutil
-import collections
 import sys
-import platform
 import xml.etree.ElementTree as etree
 from xml.dom import minidom
-import util
 
 DosGame = collections.namedtuple('DosGame',
                                  'dosname metadataname name genres publisher developer year frontPic manual desc')
 
 
+class GameGenres(Enum):
+    # the target genres we want to map to
+    UNKNOWN             = "Unknown"
+    MISC                = "Misc"
+    ACTION_ADVENTURE    = "Action-Adventure"
+    ADVENTURE_VISUAL    = "Adventure-Visual"
+    ADVENTURE_TEXT      = "Adventure-Text"
```

Comment:
I don't want tow different genres for adventure games, so let's stick with Adventure-Visual

Reply:
Sure, no problem. 
Just wondering, why "Adventure-Visual" and not just "Adventure"?  (as it includes text)

Reply:
for Visual Novel